!!python/object/new:dynamic_reconfigure.encoding.Config
dictitems:
  blob_filter_by_area: true
  blob_filter_by_circularity: false
  blob_filter_by_convexity: false
  blob_filter_by_inertia: false
  blob_max_area: 10000.0
  blob_max_circularity: 1.0
  blob_max_convexity: 1.0
  blob_max_inertia: 1.0
  blob_min_area: 200.0
  blob_min_circularity: 0.0
  blob_min_convexity: 0.0
  blob_min_inertia: 0.0
  groups: !!python/object/new:dynamic_reconfigure.encoding.Config
    dictitems:
      blob_filter_by_area: true
      blob_filter_by_circularity: false
      blob_filter_by_convexity: false
      blob_filter_by_inertia: false
      blob_max_area: 10000.0
      blob_max_circularity: 1.0
      blob_max_convexity: 1.0
      blob_max_inertia: 1.0
      blob_min_area: 200.0
      blob_min_circularity: 0.0
      blob_min_convexity: 0.0
      blob_min_inertia: 0.0
      groups: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      id: 0
      name: Default
      outline_scale_factor: 50
      p_bot_left_x: 299
      p_bot_left_y: 287
      p_bot_right_x: 956
      p_bot_right_y: 287
      p_top_left_x: 490
      p_top_left_y: 126
      p_top_right_x: 765
      p_top_right_y: 126
      parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      parent: 0
      pattern: 2
      pattern_delta_x: 0.08
      pattern_delta_y: 0.08
      pattern_offset_x: 0.453
      pattern_offset_y: -0.269
      points_per_col: 7
      points_per_row: 4
      state: true
      type: ''
    state: []
  outline_scale_factor: 50
  p_bot_left_x: 299
  p_bot_left_y: 287
  p_bot_right_x: 956
  p_bot_right_y: 287
  p_top_left_x: 490
  p_top_left_y: 126
  p_top_right_x: 765
  p_top_right_y: 126
  pattern: 2
  pattern_delta_x: 0.08
  pattern_delta_y: 0.08
  pattern_offset_x: 0.453
  pattern_offset_y: -0.269
  points_per_col: 7
  points_per_row: 4
state: []
